runtime/drivers/snowflake/sql_store.go

```diff
@@ -86,7 +86,7 @@ func (c *connection) QueryAsFiles(ctx context.Context, props map[string]any) (dr
 		return nil, drivers.ErrNoRows
 	}
 
-	tempDir, err := os.MkdirTemp(c.configProperties.TempDir, "snowflake")
+	tempDir, err := c.storage.RandomTempDir("snowflake")
```

Comment:
Is `"snowflake"` needed here? Thinking the `c.storage` will already have `"snowflake"` as a prefix?

Reply:
This is a random directory within parent `instance_id/snowflake` where temporarily download parquet files will be written.
Different sources can be ingesting data in parallel so need a unique directory for each.
Though as of now only one file is generated but the rest of the code is structured to support multiple files and hence ingesting them in a unique directory and later removing the directory seems like a safer option.

Reply:
We can also name it after the sink table but will need to pass it here.

Reply:
Okay makes sense, no need to change